sentinelhub/config.py

```diff
@@ -10,11 +10,16 @@
 from pathlib import Path
 from typing import Dict, Iterable, Optional, Tuple, Union
 
+import tomli
+import tomli_w
+
+DEFAULT_PROFILE = "default"
+
 
 class SHConfig:  # pylint: disable=too-many-instance-attributes
     """A sentinelhub-py package configuration class.
 
-    The class reads the configurable settings from ``config.json`` file on initialization:
+    The class reads the configurable settings from ``config.toml`` file on initialization:
```

Comment:
There are still a bunch of instances of mentions of `config.json` in the repo, did you plane to change these parts as well?

Reply:
there is an open issue for adjusting docs, want to wait until we solidify everything